minter-suite/seadrop-integration.md

```diff
@@ -0,0 +1,92 @@
+---
+order: -20
+---
+
+# SeaDrop Integration
+
+This page provides an overview of an integration solution for the Art Blocks Shared Minter Suite with SeaDrop. This is a special integration that allows Art Blocks projects to be minted and sold on OpenSea via SeaDrop, and is not a standard integration with the Shared Minter Suite.
+
+The integration requires a few non-standard steps to ensure that the SeaDrop contract can interact with the minter suite. The steps below outline the process for integrating SeaDrop with the Art Blocks Shared Minter Suite.
+
+---
+
+## Integration Process
+
+The integration process follows the custom, one-off minter steps, and supports minting on any project that uses the Shared Minter Suite.
+
+### 1. Artist configures their project
+
+The artist configures their project as they normally would, using the Art Blocks Creator Dashboard.
```

Comment:
add a link to docs `https://docs.artblocks.io/`

Reply:
assuming you meant `https://create.artblocks.io` - updated in [142c186](https://github.com/ArtBlocks/artblocks-docs/pull/314/commits/142c1866a6bf1f6cad6189f26aef34be0023f256)